examples/nlp/language_modeling/speechllm_multitask_dataprep.py

```diff
@@ -88,7 +88,7 @@ def __init__(
             self.add_context_records_to_manifest()
         
         self.base_data_dir = get_base_dir([item["audio_filepath"] for item in self.data])
-        self.filter_invalid_records()
+        # self.filter_invalid_records()
```

Comment:
Why is this removed?

Reply:
The filter_invalid_records function was added because some LibriTTS audio files (5-10) on my machine were corrupt. But it takes a while to filter invalid records for other datasets (HiFiTTS, Riva etc) which don't have this issue. So I have commented it for now.